aten/src/ATen/native/cuda/Loops.cuh

```diff
@@ -9,6 +9,12 @@ namespace at { namespace native {
 constexpr int num_threads = C10_WARP_SIZE * 2;
 constexpr int thread_work_size = 4;
 constexpr int block_work_size = thread_work_size * num_threads;
+// Below is a macro version of the same thing, this is needed because
```

Comment:
Please put them above and do something like: `constexpr int num_threads = NUM_THREADS;` to ensure we never have them defined differently, thanks.

Reply:
fixed